checkpointctl.go

```diff
@@ -4,15 +4,23 @@ package main
 
 import (
 	"fmt"
+	"log"
 	"os"
+	"path/filepath"
 
 	metadata "github.com/checkpoint-restore/checkpointctl/lib"
 	"github.com/spf13/cobra"
 )
 
 var (
-	name    string
-	version string
+	name      string
+	version   string
+	format    string
+	stats     bool
+	mounts    bool
+	fullPaths bool
```

Comment:
Since `checkpointctl inspect` is using a more compact format to display checkpoint information (i.e., `tree` or `json`), I think we could remove the `--full-paths` option.